src/app/entity/EventRequest.ts

```diff
@@ -0,0 +1,30 @@
+import {CompleteBetType} from "./CompleteBetType";
+import {Status} from "./Status";
+
+export class EventRequest {
```

Comment:
For consistency, I recommend the data models be either all `class` or all `interface` (for example, `CompleteBetType` is an interface). I generally think that interfaces are easier to work with, so please change this and any other model classes in this PR to interface